server/static/gateway.css

```diff
@@ -0,0 +1,191 @@
+
+/* HEADER NAVIGATION */
+.navbar {
+	min-height: 48px;
+	padding: 0 12px;
+	border-radius: 0;
+}
+
+.navbar-default {
+	border-color: rgba(0,0,0,0);
+	background-color: #fff;
+	box-shadow: 0 3px 4px 0 rgba(0,0,0,.2), 0 3px 3px -2px rgba(0,0,0,.14), 0 1px 8px 0 rgba(0,0,0,.12);
+}
+.navbar-default .navbar-brand,
+.navbar-default .navbar-brand:hover,
+.navbar-default .navbar-text,
+.navbar-default .navbar-nav>li>a {
+	color: rgba(0,0,0,0.87);
+}
+
+.navbar-default .navbar-brand {
+	padding: 14px 15px;
+}
+.navbar-default .navbar-nav>li>a {
+	text-transform: uppercase;
+	font-size: 13px;
+}
+.navbar-default .navbar-nav>li>a:hover {
+	background-color: transparent;
+}
+.navbar-default .navbar-form {
+	margin: 10px 0;
+}
+.login-link,
+.login-link:hover {
+	font-size: 12px;
+	padding: 5px 10px;
+	text-decoration: none;
+	color: rgba(0,0,0,0.87);
+	border-radius: 4px;
+	border: 1px solid rgba(0,0,0,.26);
+}
+
+/* MAIN CONTENT AREA */
+body {
+	font-family: "Roboto","Helvetica","Arial",sans-serif;
+}
+
+.container {
+	padding: 0;
+}
+
+h3 {
+	color: rgba(0,0,0,.87);
+    font-family: "Roboto","Helvetica","Arial",sans-serif;
+    font-size: 20px;
+    font-weight: 500;
+    letter-spacing: normal;
+    line-height: 28px;
+    margin-bottom: 16px;
+    margin-top: 0;
+}
+.container h3 {
+    height: auto;
+	padding: 0 8px 0 0;
+	display: inline-block!important;
+	vertical-align: middle;
+}
+.btn {
+	font-size: 12px;
+	padding: 5px 10px;
+}
+.btn-primary,
+.btn-primary:hover {
+	background-color: #4284f4;
+	border-color: transparent;
+}
+.btn-default,
+.btn-default:hover {
+	color: rgba(0,0,0,.87);
+	background-color: #fff;
+	border-color: rgba(0,0,0,.26);
+}
+.glyphicon {
+	display: none;
+}
+.welcomepage {
+	width: 745px;
+	margin: 40px auto;
+}
+
+/* TABLE */
+.addservice {
+	margin-bottom: 16px;
+}
+.container div ul {
+	margin: 32px 0 10px 0;
+	padding: 0;
+}
+.container div ul li {
+	list-style: none;
+}
+.container div ul li h4 {
+	display: inline;
+	vertical-align: middle;
+	padding: 0 8px 0 0;
+}
+.table tr th {
+	font-size: 12px;
+}
+.table tr td {
+	font-size: 12px;
+	background-color: rgba(250,250,250,1);
+}
+.table tr.accounts td {
+	padding-left: 24px;
+	background-color: rgba(255,255,255,1);
```

Comment:
nit: your preference. (255,255,255,1) is white. Alternatively you can just say background-color: white;